### DISCUSSION TITLE
Old irrelevant events in my feed

### DISCUSSION THREAD

**Discussion by xonixx** [community/community#66215] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Why does it show me something happened 2 months ago?
The whole value of old feed for me was that I can see what new happened during the last day.
![18D70119-C4E4-4719-BADC-9AF728015E41](https://github.com/community/community/assets/11706893/42e68512-4bac-4a2e-a3c7-b67947ba8654)

**Comment by beeman** [comment, marked as answer]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Why does it show me something happened 2 months ago? The whole value of old feed for me was that I can see what new happened during the last day. ![18D70119-C4E4-4719-BADC-9AF728015E41](https://user-images.githubusercontent.com/11706893/266156798-42e68512-4bac-4a2e-a3c7-b67947ba8654.jpeg)

The Product Managers don't use GitHub. So they don't have any clue...
